src/contexts/Drive/Drive.context.tsx

```diff
@@ -88,14 +157,23 @@ export const DriveContextProvider: React.FC<DriveContextProviderProps> = ({ chil
    * 3. Network
    *
    */
+
   const loadFolderContent = async (folderId: number, options?: LoadFolderContentOptions) => {
     const shouldPullFromCache = options?.pullFrom ? options?.pullFrom.includes('cache') : true;
     const shouldPullFromNetwork = options?.pullFrom ? options?.pullFrom.includes('network') : true;
+
+    // Check if the items have been modified from another platform
```

Comment:
Use comments only when they add something extra to the reader, your code should be self explanatory, `getModifiedDriveItems` is a name good enough to understand what happens, how it happens is not needed to be defined here, instead that's inside the function definition.

You can also use TSDoc in the function if you want to add more info